fdbserver/DiskQueue.actor.cpp

```diff
@@ -555,6 +561,7 @@ class RawDiskQueue_TwoFiles : public Tracked<RawDiskQueue_TwoFiles> {
 		}
 	}
 
+	// Q: From where this function is called, its return seems to be lastPageData, instead of both first and last page data?
```

Comment:
Maybe figure out the answer to this question?

Reply:
Good catch! I will update it after figuring it out. ;)